apps/web/components/apps/App.tsx

```diff
@@ -14,11 +13,6 @@ const ShellHeading = () => {
 export default function WrappedApp(props: AppPageProps) {
   return (
     <Shell smallHeading isPublic hideHeadingOnMobile heading={<ShellHeading />} backPath="/apps" withoutSeo>
-      <HeadSeo
-        title={props.name}
-        description={props.description}
-        app={{ slug: props.logo, name: props.name, description: props.description }}
-      />
```

Comment:
this is replaced by `generateMetadata` in `apps/[slug]/page.tsx` where we use `generateAppMetadata`